### PR TITLE
Make Upgrade page comprehensive  (draft for discussion)

### PR DESCRIPTION
**Pull request by StackScribe**
This PR creates an "Upgrade" page that covers all releases.
* Starts with generic instructions for upgrading control and execution planes
* Includes notes for upgrading multi-cluster instances
* Provides release-specific upgrade information for recent releases
* Includes a table that links to release notes and release-specific upgrade instructions for older releases

This is a draft for review and discussion.

https://github.com/keptn/keptn.github.io/issues/1435

### CODE REVIEW

File: content/docs/install/upgrade/index.md [hunk 1]
```diff
@@ -4,6 +4,65 @@ description: Upgrade your Keptn to 0.17
...
 
+Keptn only supports upgrading from one release to the next;
+you cannot skip releases.
+This page
+
+* Summarizes the upgrade process for the Keptn control plane and Keptn execution plane
+* Includes some notes about special steps required when updating multi-cluster Keptn instances
+* Gives release-specific notes about upgrading from recent releases
+* Includes links to Release Notes and upgrade instructions for older releases,
+  to help customers who must upgrade from older releases
```
Comment (mowies) [inline content/docs/install/upgrade/index.md:15]: Maybe add links to the corresponding sections here?

Comment (StackScribe) [reply content/docs/install/upgrade/index.md]: Good idea!  Done.

File: content/docs/install/upgrade/index.md [hunk 2]
```diff
@@ -4,7 +4,68 @@ description: Upgrade your Keptn to 0.17
...
+* For Keptn releases 0.15.x and later,
+you may need to set the `K8S_DEPLOYMENT_NAME` environment variable on each execution plane to a unique name.
+* [Add note about setting NATS_URL env variable]
+
+
+## Upgrade notes by release
+
+### Upgrade from Keptn 0.18.x to Keptn 0.19.x
+
+### Upgrade from Keptn 0.17.x to Keptn 0.18.x
```
[the remark this answers is not in the document]
Comment (StackScribe) [reply content/docs/install/upgrade/index.md:124]: Done

**Review by oleg-nenashev — APPROVED** [review]
No objections

**Pull request merged**